src/components/interlocutor/screens/InterlocutorList/actions/index.js

```diff
@@ -17,27 +17,48 @@ export function getInterlocutorList() {
 }
 
 export function getInterlocutorListFromRequest(request) {
-  if ( request.length > 0 ) {
-    return (dispatch, getState) => executeSql(
-      `SELECT name, link_to_me, id 
+  if (request.length > 0) {
+    requestList = request.split(' ');
+    return (dispatch, getState) => {
+      let promises = [];
+      request = `SELECT name, link_to_me, id 
```

Comment:
You're using the function parameter that you reassign ... Maybe you've interverted `request` and `requestList` ?

Reply:
requestSql added